<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>308</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Class Informer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QtResource.qrc">
    <normaloff>:/classinf/icon.png</normaloff>:/classinf/icon.png</iconset>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>275</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox1">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>95</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Place structures</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>125</y>
     <width>256</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Process static initializers &amp;&amp; terminators</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox3">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>155</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Overwrite anterior comments</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>185</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Audio on completion</string>
   </property>
  </widget>
   <widget class="QCheckBox" name="checkBox5">
     <property name="geometry">
       <rect>
         <x>15</x>
         <y>215</y>
         <width>151</width>
         <height>17</height>
       </rect>
     </property>
     <property name="font">
       <font>
         <family>Noto Sans</family>
         <pointsize>10</pointsize>
       </font>
     </property>
     <property name="toolTip">
       <string notr="true"/>
     </property>
     <property name="text">
       <string>Clean</string>
     </property>
   </widget>
   <widget class="QCheckBox" name="checkBox6">
     <property name="geometry">
       <rect>
         <x>15</x>
         <y>245</y>
         <width>151</width>
         <height>17</height>
       </rect>
     </property>
     <property name="font">
       <font>
         <family>Noto Sans</family>
         <pointsize>10</pointsize>
       </font>
     </property>
     <property name="toolTip">
       <string notr="true"/>
     </property>
     <property name="text">
       <string>Full Clear</string>
     </property>
   </widget>
   <widget class="QLabel" name="linkLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>280</y>
     <width>136</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>&lt;a href=&quot;http://www.macromonkey.com/bb/index.php/topic,13.0.html&quot; style=&quot;color:#AA00FF;&quot;&gt;Class Informer Fourm&lt;/a&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>292</width>
     <height>74</height>
    </rect>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="QtResource.qrc">:/classinf/banner.png</pixmap>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="versionLabel">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>45</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Version: 2.0
By Sirmabus</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="QtResource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
